launch_ros/launch_ros/actions/node.py

```diff
@@ -54,12 +54,51 @@
 from rclpy.validate_namespace import validate_namespace
 from rclpy.validate_node_name import validate_node_name
 
+import importlib_metadata
+from packaging.version import Version
 import yaml
 
 from ..descriptions import Parameter
 from ..descriptions import ParameterFile
 
 
+class NodeActionExtension:
+    """
+    The extension point for launch_ros node action extensions.
+
+    The following properties must be defined:
+    * `NAME` (will be set to the entry point name)
+
+    The following methods may be defined:
+    * `command_extension` to extend the command used to launch the node's
```

Comment:
@gbiggs nit: consider moving documentation to each method, and only listing which ones are required here.

Reply:
Done in 6754e9a366589dec1b1535eeef2e08ef765b1301.